Sources/xcproj/AEXML+XcodeFormat.swift

```diff
@@ -0,0 +1,80 @@
+import Foundation
+import AEXML
+
+extension AEXMLDocument {
+
+    var xmlXcodeFormat: String {
+        var xml = "<?xml version=\"\(options.documentHeader.version)\" encoding=\"\(options.documentHeader.encoding)\"?>\n"
+        xml += root._xmlXcodeFormat
+        return xml
+    }
+
+}
+
+extension AEXMLElement {
+
+    fileprivate var _xmlXcodeFormat: String {
```

Comment:
nitpick: No need to prefix this with an underscore. The access modifier takes care of privacy, and this codebase doesn't have underscores

Reply:
unfortunately it's either this or a different name here or in document extension, because otherwise it will try to override (document is a subclass of element), but Swift does not allow overriding in extensions

Reply:
Yeah ok, sorry. What's there is fine then